Lesson1/Task2/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter width: ");
+            int width = Convert.ToInt32(Console.ReadLine());
+
+            Console.Write("Enter height: ");
+            int height = Convert.ToInt32(Console.ReadLine());
+
+            Console.Write("Enter length: ");
+            int length = Convert.ToInt32(Console.ReadLine());
+
+            var result = Math.Sqrt((Math.Pow(width,2)+Math.Pow(height,2)+Math.Pow(length,2)));

```

Comment:
@okuzmina to make code clear, please add empty spaces, something like: 
Math.Pow(width, 2) + Math.Pow(height, 2) + Math.Pow(length, 2)